### PR TITLE
COMMERCE-12060 Fixed options field value for bundle products

### PR DESCRIPTION
**Pull request by pietrobocale**
Base PR: https://github.com/liferay-commerce/liferay-portal/pull/4184

https://liferay.atlassian.net/browse/COMMERCE-12060

### CODE REVIEW

**Comment by pietrobocale** [comment]
ci:test:sf

**Comment by pietrobocale** [comment]
ci:test:relevant

**Comment by pietrobocale** [comment]
ci:test:commerce-functional

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceOrderDisplayPageTemplates.testcase [hunk 1]
```diff
@@ -332,6 +332,97 @@ definition {
 		}
 	}
 
+	@description = "This is a test to check the COMMERCE-12060. I want to be able to use the Order Item fragment for displays correctly the options for the product bundles"
```
Comment (22LucaGiannotta22) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceOrderDisplayPageTemplates.testcase:335]: As an admin I want the Order Item fragment to display correctly the options for the product bundles

**Comment by pietrobocale** [comment]
ci:test:sf

**Comment by pietrobocale** [comment]
ci:test:relevant

**Comment by pietrobocale** [comment]
ci:forward

**Pull request closed without merging**